PC/msvcrtmodule.c

```diff
@@ -475,6 +491,8 @@ msvcrt_set_error_mode_impl(PyObject *module, int mode)
 }
 #endif /* _DEBUG */
 
+#ifndef MS_WINDOWS_NON_DESKTOP
```

Comment:
```suggestion
#ifndef MS_WINDOWS_GAMES
```

In 10.0.19041, `GetErrorMode()` is available for the app and system API partitions, but it's not available for games. `SetErrorMode()` is available for the games partition.